app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -66,7 +66,7 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         for (int i = 0; i < 5; i++) {
             RequestOff requestOff = new RequestOff();
             requestOff.setCreatedAt("16/06/2017");
-            requestOff.setStatus(0);
+            requestOff.setStatus("approved");
```

Comment:
ERROR: [WrongConstant] Must be one of: StatusCode.ACCEPT_CODE, StatusCode.PENDING_CODE, StatusCode.REJECT_CODE
 Ensures that when parameter in a method only allows a specific set of constants, calls obey those rules.